docs/admin/federation/kubefed.md

```diff
@@ -0,0 +1,172 @@
+---
+assignees:
+- madhusudancs
+
+---
+Kubernetes v1.5 introduced a new command line tool called `kubefed`
+to facilitate the administration of cluster federations. `kubefed`
+stands for "Kubernetes Federate". It aids in both deploying a
+Kubernetes Cluster Federation control plane and adding/removing
+clusters from it. 
+
+This guide explains how to administer a Kubernetes Cluster Federation
+using `kubefed`.
+
+`kubefed` is considered alpha in Kubernetes v1.5.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes cluster. Please
+see one of the [getting started](/docs/getting-started-guides/) guides
+for installation instructions for your platform.
+
+## Getting `kubefed`
+
+Download the Kubernetes client tarball corresponding to a release
+[from the release page](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md),
+extract the binaries in the tarball to one of the directories
+in your `$PATH` and set the executable permission on those binaries.
+
+```shell
+curl -O https://storage.googleapis.com/kubernetes-release/release/v1.5.0/kubernetes-client-linux-amd64.tar.gz
+tar -xzvf kubernetes-client-linux-amd64.tar.gz
+sudo cp kubernetes/client/bin/kubefed /usr/local/bin
+sudo chmod +x /usr/local/bin/kubefed
+sudo cp kubernetes/client/bin/kubectl /usr/local/bin
+sudo chmod +x /usr/local/bin/kubectl
+```
+
+## Choosing a host cluster.
+
+Choose one of your Kubernetes clusters as a cluster where you want to
+host the federation control plane components. We refer to this cluster
+as a "host cluster" henceforth. Ensure that you have a kubeconfig entry
+in your local kubeconfig corresponding to this cluster. You can verify
+this by running:
+
+```shell
+kubectl config get-contexts
+```
+
+and ensuring that there is a context corresponding to your host
```

Comment:
Can you show an example of successful output?

"The output should contain a context corresponding to your host cluster, similar to the following:"